conda_self/package_info.py

```diff
@@ -0,0 +1,59 @@
+import configparser
+from pathlib import Path
+
+
+class MultipleDistInfoDirsFound(Exception):
+    pass
+
+
+class NoDistInfoDirFound(Exception):
+    pass
```

Comment:
These should inherit from `conda.exceptions.CondaError` to avoid ugly tracebacks unless in verbose mode.